contribution/words.json

```diff
@@ -383,6 +395,7 @@
 		"Unhoused Person",
 		"Vigilante",
 		"VTuber",
+		"Vitrual Reality Goggles",
```

Comment:
Vi**rt**ual instead of Vi**tr**ual

Reply:
Thank you man! I feel like contributing more to this wonderful game.